include/fastrtps/serializer/serializer.h

```diff
@@ -0,0 +1,502 @@
+#include <fastrtps/types/DynamicTypePtr.h>
+#include <fastrtps/types/TypeObjectHashId.h>
+#include <fastrtps/types/TypeObject.h>
+namespace eprosima{
+
+namespace fastcdr{
+    class Cdr ;
+}
+namespace fastrtps{
+
+namespace types{
+    class DynamicData ;
+}
+
+namespace serializer{
+
+/*********************
+ *     SERIALIZER    *
+ *********************/
+	
+template<class Serializable>
+class Serializer{
+private:
+    Serializer() = delete ;
+public:
+    Serializer<Serializable>(Serializable *p):p_(p){}
+    
+    void serialize(eprosima::fastcdr::Cdr& cdr)
+				{
+        serialize(p_, cdr) ;
+    }
+    void deserialize(eprosima::fastcdr::Cdr& cdr)
+				{
+        deserialize(p_, cdr) ;
+    }
+    size_t getCdrSerializedSize(size_t current_alignment)
+				{
+				    return getCdrSerializedSize( *p_,current_alignment) ;
+				}
+
+private:
+// FATTI PER ORA
+
+/*DynamicData Methods*/
+    void serialize(eprosima::fastrtps::types::DynamicData *dd, eprosima::fastcdr::Cdr& cdr) const ;
```

Comment:
Coding style:
Each parameter in a new line and methods separated by a blank line.

Review all code for this kind of issues.